### ISSUE TITLE
Bypass pop-up on tablet mode

### ISSUE THREAD

**Issue by RobbieNor** [RobbieNor/indicator-acertouchscreen#5]
On tablet mode, pop-up asks about enabling Gnome Accessibility. This should be dealt with automatically. Can't see any harm in enabling this. Disabling accessibility automatically is out of the question.

**Comment by RobbieNor** [comment]
This doesn’t seem to be happening on my system anymore. I'm unsure why, may be worth trying indicator out on a fresh install or VM to find cause and proper solution.
